src/core/retargeting_engine/python/deviceio_source_nodes/pedals_source.py

```
@@ -95,24 +100,25 @@ def output_spec(self) -> RetargeterIOType:
-        Convert DeviceIO Generic3AxisPedalOutput to standard Generic3AxisPedalInput tensor.
+        Convert DeviceIO Generic3AxisPedalOutputTrackedT to standard Generic3AxisPedalInput tensor.
 
         Calls ``set_none()`` on the output when pedal data is inactive.
 
         Args:
-            inputs: Dict with "deviceio_pedals" containing Generic3AxisPedalOutput at index 0
+            inputs: Dict with "deviceio_pedals" containing Generic3AxisPedalOutputTrackedT wrapper
             outputs: Dict with "pedals" OptionalTensorGroup
         """
-        pedal: "Generic3AxisPedalOutput" = inputs["deviceio_pedals"][0]
+        tracked: "Generic3AxisPedalOutputTrackedT" = inputs["deviceio_pedals"][0]
```

Ruff (0.15.2)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)